Adding fallback CSS vars example (#17559)
mdn/content: files/en-us/web/css/var/index.md

}
```

### Using a custom property as a fallback

```css
:root {
  --backup-bg-color: teal;
}

body {

>   /* main-bg-color isn't set, it will fallback on backup-bg-color. If backup-bg-color isn't set it will fallback on white. */

Reviewer: ```suggestion
  /* main-bg-color isn't set, it will fall back to backup-bg-color. If backup-bg-color isn't set it will fall back to white. */
```
